src/app/components/case/correspondence/CorrespondenceForm/scaffold.tsx

```diff
@@ -25,27 +25,28 @@ export default (caseId: Components.Schemas.Case["id"], correspondences: MockComp
         isRequired: true
       }
     },
-    submit: {
-      type: "SubmitButton",
-      props: {
-        label: "Notitie verwerken"
-      }
-    },
     secondaryButton: {
       type: "Button",
       props: {
         label: "Annuleren",
         variant: "primaryInverted",
-        onClick: () => navigateTo(`/zaken/${ caseId }`)
+        onClick: () => navigateTo("/zaken/:id", { id: caseId })
```

Comment:
Why is this syntax better?

Reply:
Otherwise a "route does not exist" warning will show up in the logs